NEWS.md

```diff
@@ -5,6 +5,10 @@
 
 ## (current master)
 
+## 1.25.0
+- drop support for julia `1.5`
+- support julia LTS `1.6` and julia `1.7`
+
```

Comment:
Since we got TagBot, we don't update this file anymore, since the release note summaries are quite good.